packages/toolkit/src/view/knowledge/KnowledgeBaseView.tsx

```diff
@@ -56,6 +57,7 @@ export const KnowledgeBaseView = (props: KnowledgeBaseViewProps) => {
   const { accessToken, enabledQuery, selectedNamespace } = useInstillStore(
     useShallow(selector),
   );
+  const isLocalEnvironment = env("NEXT_PUBLIC_API_GATEWAY_URL");
```

Comment:
@thewbuk This is not safe, you can use env("NEXT_PUBLIC_APP_ENV") === "CE" to identify ce environment